autoload/importjs.vim

```diff
@@ -21,6 +21,11 @@ endfunction
 " Execute the command. If we get an empty response, keep trying to execute until
 " we get a non-empty response or hit the max number of tries.
 function importjs#TryExecPayload(payload, tryCount)
+  if !exists("g:ImportJSChannel")
+    echoerr "If you just added `package.json` just wait a few seconds"
```

Comment:
I don't think I understand this message... What does package.json have to do with it?

Reply:
This is due to checking for a `package.json` file every couple of seconds when idle if the importjs process hasn't started yet. When you start vim, if your pwd contains a `package.json` file it starts importjs. If not - it checks on `CursorHold/I` if you have created the file.

Reply:
I see. So this is a way to prevent importjs from starting in projects where there is no suitable root folder? I find the message confusing, maybe it's better to mention that importjs can't run in the current context?

Reply:
ok